index.js

```diff
@@ -328,13 +315,9 @@ NYC.prototype.reset = function () {
 }
 
 NYC.prototype._wrapExit = function () {
-  var _this = this
-
   // we always want to write coverage
   // regardless of how the process exits.
-  onExit(function () {
-    _this.writeCoverageFile()
-  }, { alwaysLast: true })
+  onExit(() => { this.writeCoverageFile() }, { alwaysLast: true })
```

Comment:
Could you add the line-feeds back in?  I like that you switched to arrow function but prefer that single-line only be used for arrow expressions like `() => value` or empty functions like `() => {}`.

Reply:
I was maybe a little over eager there, change made